certs/service.go

```diff
@@ -34,35 +35,33 @@ var _ Service = (*certsService)(nil)
 //go:generate mockery --name Service --output=./mocks --filename service.go --quiet --note "Copyright (c) Abstract Machines"
 type Service interface {
 	// IssueCert issues certificate for given thing id if access is granted with token
-	IssueCert(ctx context.Context, token, thingID, ttl string) (Cert, error)
+	IssueCert(ctx context.Context, token, thingID, ttl string) (sdk.Certificate, error)
```

Comment:
Certs should have it own Type, 

So please convert it magistrala certs object.   
I mean please revert back to original 
In `amcerts` convert sdk.Certificate to magistrala.Cert